eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/SubmittedVariantSplittingPolicy.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2022 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.accession.clustering.batch.io;
+
+import uk.ac.ebi.eva.accession.core.model.eva.SubmittedVariantEntity;
+import uk.ac.ebi.eva.accession.core.summary.SubmittedVariantSummaryFunction;
+
+import java.util.Objects;
+
+/**
+ * This class represents the policy for choosing which SS ID to keep when 2 of them are split.
+ */
+public class SubmittedVariantSplittingPolicy {
+
+    public static class SplitDeterminants {
+        private final SubmittedVariantEntity submittedVariantEntity;
+        private final String ssHash;
+        private final Long rsID;
+
+        public SplitDeterminants(SubmittedVariantEntity submittedVariantEntity, String ssHash,
+                                 Long rsID) {
+            this.submittedVariantEntity = submittedVariantEntity;
+            this.ssHash = ssHash;
+            this.rsID = rsID;
+        }
+
+        public SubmittedVariantEntity getSubmittedVarianEntity() {
+            return submittedVariantEntity;
+        }
+
+        public String getSsHash() {
```

Comment:
```suggestion
        public String getSSHash() {
```
just for consistency with the rest of the codebase

Reply:
Will address..